README.md

```diff
@@ -2,9 +2,10 @@
 <img width="300" src="https://github.com/shipt/osmosis/blob/master/logo.png" />
 </p>
 
-Osmosis utilizes React context and allows you to create your own custom hooks to provide lightweight and modularized global state management for any React or React Native project.
+Osmosis utilizes React context and allows you to create your own custom hooks to provide a lightweight and modularized global state management for any React or React Native project.
```

Comment:
Grammatically i think this was correct as-is, but could definitely be cleaned up a bit....how about something like 

"to create your own custom hooks to provide a lightweight and modularized solution for global state management"

Reply:
sounds good.